src/lwc/utilPageLevelMessage/utilPageLevelMessage.js

```diff
@@ -65,7 +66,7 @@ export default class utilPageLevelMessage extends LightningElement {
     }
 
     get subtitleClass() {
-        let classNames = 'slds-p-top_x-small ';
+        let classNames = (this.subtitleClassOverride || 'slds-p-top_x-small') + ' ';
```

Comment:
@psadie-sfdo , can you please review this change to ensure it does not affect GE components?

Reply:
Looks from GE components!